src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/AccessibleObjects/DataGridViewRowAccessibleObjectTests.cs

```diff
@@ -2379,6 +2380,26 @@ public void DataGridViewRowAccessibleObject_GetPropertyValue_ValueValuePropertyI
         Assert.False(dataGridView.IsHandleCreated);
     }
 
+    [WinFormsFact]
+    public void DataGridView_SwitchConfigured_AdjustsRowAndColumnStartIndices()
+    {
+        var originalSwitchValue = LocalAppContextSwitches.DataGridViewRowStartsAtOne;
+        if (!originalSwitchValue)
+        { LocalAppContextSwitches.Test_SetDataGridViewRowStartsAtOne(true); }
```

Comment:
just for readability 
```suggestion
        {
            LocalAppContextSwitches.Test_SetDataGridViewRowStartsAtOne(true); 
        }
```